.drone.yml

```diff
@@ -120,28 +120,6 @@ platform:
   arch: amd64
 
 steps:
-- name: manifest-runtime
-  image: rancher/hardened-build-base:v1.16.4b7
-  commands:
-  - docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
-  - DRONE_TAG=${DRONE_TAG} make publish-manifest-runtime
-  environment:
-    DOCKER_PASSWORD:
-      from_secret: docker_password
-    DOCKER_USERNAME:
-      from_secret: docker_username
-  volumes:
-  - name: docker
-    path: /var/run/docker.sock
-  when:
-    event:
-    - tag
-    instance:
-    - drone-publish.rancher.io
-    ref:
-    - refs/head/master
-    - refs/tags/*
-
 - name: manifest-kubernetes
```

Comment:
we should also remove the manifest-kubernetes since its already being made in https://github.com/rancher/image-build-kubernetes/blob/master/.drone.yml#L61-L76

Reply:
Done